Dockerfile

```diff
@@ -29,8 +29,9 @@ ENV XDG_CONFIG_HOME /usr/local/share/ocrd-resources
 
 WORKDIR /build/ocrd_pagetopdf
 COPY . .
+COPY ocrd-tool.json .
 # prepackage ocrd-tool.json as ocrd-all-tool.json
-RUN ocrd ocrd-tool ocrd_pagetopdf/ocrd-tool.json dump-tools > $(dirname $(ocrd bashlib filename))/ocrd-all-tool.json
+RUN ocrd ocrd-tool ocrd-tool.json dump-tools > $(dirname $(ocrd bashlib filename))/ocrd-all-tool.json
```

Comment:
Dang, I did not do that for ocrd_froc 😨

Reply:
It's never too late (for a post-release) ;-)

Reply:
https://github.com/OCR-D/ocrd_froc/pull/20

Reply:
thanks, merged, released.